Core/src/main/java/com/braintreepayments/api/internal/SignatureVerification.java

```diff
@@ -31,6 +36,7 @@
      * @param publicKeyHashCode the hash code of the app's public key.
      * @return true is signature is valid or signature verification has been disabled.
      */
+    @Deprecated
```

Comment:
Can you provide a deprecation note in Android (I know you can in iOS). Something to signal that this won't work in Android 12?

Reply:
Just checked and unfortunately this isn't possible with the deprecated tag in Android